packages/near-fast-auth-signer/src/lib/controller.ts

```diff
@@ -146,13 +155,31 @@ class FastAuthController {
 
   async signDelegateAction({ receiverId, actions, signerId }) {
     this.assertValidSigner(signerId);
-
-    const account = new Account(this.connection, this.accountId);
-    return account.signedDelegate({
-      actions,
-      blockHeightTtl: 60,
-      receiverId,
-    });
+    try {
+      // webAuthN supported browser
+      const account = new Account(this.connection, this.accountId);
+      return account.signedDelegate({
+        actions,
+        blockHeightTtl: 60,
+        receiverId,
+      });
+    } catch {
+      // fallback, non webAuthN supported browser
+      // @ts-ignore
+      const oidcToken =  firebaseAuth.currentUser.accessToken;
```

Comment:
Instead of disabling the typescript can we use the user.getIdToken() ?

I logged it a few times on different scenarios and looks like it returns the same token as the accessToken.

I tried to find the docs for the accessToken and check if it's idd the same as user.getIdToken() but I can't find the docs for accessToken